src/test/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocumentIntegrationTest.java

```diff
@@ -91,6 +93,12 @@ public void testToCopy() throws WorkflowException {
         assertNull(cuDisbursementVoucherDocument.getCancelDate());
         assertEquals(KFSConstants.DocumentStatusCodes.INITIATED, cuDisbursementVoucherDocument.getFinancialSystemDocumentHeader().getFinancialDocumentStatusCode());
 
+        MessageList messageList = KNSGlobalVariables.getMessageList();
+        System.err.println("KNSGlobalVariables.getMessageList().size(): " + messageList.size());
```

Comment:
It's temporary to help figure out why this test fails in Jenkins but not locally for me.